webhook_to_fedora_messaging/endpoints/user.py

```diff
@@ -0,0 +1,58 @@
+from flask import Blueprint, Flask, request, Response, Request
+from ..database import db
+from ..models.user import User
+from sqlalchemy_helpers import get_or_create
+
+app = Flask(__name__)
+user_endpoint = Blueprint("user_endpoint", __name__)
+
+@user_endpoint.route("/user", methods=["POST"])
+def create_user():
+    """
+        Used for creating a new user by sending a post request to /user/ path.
+    """
+    
+    if not validate_request(request):
+        return Response("{'message': 'Bad Request'}", status=400, mimetype='application/json')
+    session = db.Session()
+    user, created = get_or_create(session, User, username=request.json['username'])
+    if not created:
+        return Response("{'message': 'User Already Exists'}", status=409, mimetype='application/json')
+    else:
+        return Response("{'message': 'Created', 'uuid': '%s'}" % user['id'], status=201, mimetype='application/json')
+        
+    
+@user_endpoint.route("/user/search", methods=["GET"])
+def get_user():
+    
+    if not validate_request(request):
+        return Response("{'message': 'Bad Request'}", status=400, mimetype='application/json')
+    session = db.Session()
+    users = session.query(User).filter(User.username.like(request.json['username'])).all()
+    if users is None or users == []:
+        return Response("{'message': 'Not Found'}", status=404, mimetype='application/json')
+    else:
+        return Response("{'user_list': "+ users +"}", status=200, mimetype='application/json')
+    
+    
+@user_endpoint.route("/user", methods=["GET"])
+def lookup_user():
+    if not validate_request(request):
+        return Response("{'message': 'Bad Request'}", status=400, mimetype='application/json')
+    session = db.Session()
+    
+    user = session.query(User).filter(User.username == request.json['username']).first()
```

Comment:
Add exception handling for `KeyError` (when the `username` key is not found in the JSON body). Abort the request with the `HTTP 429 Unprocessable Entity` error.